### ISSUE TITLE
Error: spawn npm ENOENT

### ISSUE THREAD

**Issue by montanojd** [AntlerVC/firetable#192]
![image](https://user-images.githubusercontent.com/60124037/92599930-80d62c00-f270-11ea-8495-3c5486dee4a5.png)

**Comment by dpsoccerdude101** [comment]
![error](https://user-images.githubusercontent.com/37779084/92650585-a2f49c00-f2ba-11ea-839e-e357bcd7a780.png)

**Comment by notsidney** [comment]
Hi, thanks for bringing this to our attention. This looks like a Windows-specific bug. I’ve issued a fix in v1.0.2, please let me know if you have any further issues.

**Comment by montanojd** [comment]
Hello, thanks for the answer I am going to test the installation, an
additional query, can I use this system for an existing database
in Firestore?

**Comment by shamsmosowi** [comment]
Yes, @notsidney even built an import wizard, to help with creating the tables automatically for your existing collections

**Comment by elvisun** [comment]
Still seeing this on v1.1.4

```
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: spawn yarn ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
Emitted 'error' event at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
    at onErrorNT (internal/child_process.js:415:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

**Comment by notsidney** [comment]
This should be fixed in CLI v1.1.5
